helm/values.yaml

```diff
@@ -12,12 +12,17 @@ serviceAccount:
   create: true
   name: ""
 
-## Enable authentication
+## Enable authentication (disabling till the backward compatibility is achieved)
 usePassword: false
 
 ## MongoDB admin password
 mongodbRootUsername: root
-mongodbRootPassword: ""
+mongodbRootPassword: "abrakadabra"
```

Comment:
As discussed offline. Checking in plain text password is a big NO NO from a security point of view, especially in a open source repository.
Could you please find an alternative way?